api/app/signals/apps/graphql/types.py

```diff
@@ -0,0 +1,76 @@
+from graphene import relay
+from graphene_django.filter import DjangoFilterConnectionField
+from graphene_django.types import DjangoObjectType
+
+from signals.apps.graphql.filters import (
+    CategoryFilterSet,
+    DepartmentFilterSet,
+    ServiceLevelObjectiveFilterSet,
+    StatusMessageTemplateFilterSet
+)
+from signals.apps.signals.models import (
+    Category,
+    Department,
+    ServiceLevelObjective,
+    StatusMessageTemplate
+)
+
+
+class DepartmentType(DjangoObjectType):
```

Comment:
Take this as a question, not something that must change necessarily. I'm curious as to the reasoning:

The Django Graphene Relay tutorial uses `<SomeModel>Node> naming scheme here, why deviate? (In Relay terms this is a `Node` so we could follow that naming scheme.) 

**Update:** I now see that the docs also use the `<SomeModel>Type` naming scheme elsewhere.

Reply:
It was used both in the docs, so I chose to stick with this for now.

Reply:
I am fine either way.